frontend/styles/App.scss

```diff
@@ -1,6 +1,19 @@
 @import './mixins.scss';
 @import './variables.scss';
 
+#footerAdminLinks {
+  color: $white;
+  text-align: right;
+
+  div {
+    padding: 12px 10px;
```

Comment:
Isn't spacing a version 4+ thing? We're unfortunately still using 3.3+. It's something I have to update eventually.

Reply:
Oh snap. Ok - I guess that's something for the backlog then. Sooner is probably better than later, but don't underestimate the impact of swapping to bootstrap 4. It may hit you hard.